src/deepmol/models/keras_models.py

```diff
@@ -137,15 +142,47 @@ def fit_on_batch(self, X: Sequence, y: Sequence):
         Fits model on batch of data.
         """
 
-    def reload(self) -> None:
+    @classmethod
+    def load(cls, file_path: str, **kwargs) -> 'KerasModel':
         """
         Reloads the model from disk.
+
+        Parameters
+        ----------
+        file_path: str
+            The path to load the model from.
+
+        Returns
+        -------
+        KerasModel
+            The loaded model.
         """
+        file_path_model_builder = os.path.join(file_path, 'model_builder.pkl')
+        model_builder = load_model_from_disk(file_path_model_builder)
+        file_path_model = os.path.join(file_path, 'model.h5')
+        model = keras.models.load_model(file_path_model)
+        keras_model_class = cls(model_builder=model_builder, **kwargs)
+        keras_model_class.model.model = model
+        return keras_model_class
 
-    def save(self) -> None:
+    def save(self, file_path: str = None) -> None:
```

Comment:
It is possible to also save all the attributes of the model? So when loading it is not necessary to pass the kwargs? (see comment on load) 

Maybe put it in a dictionary and save it in a pickle along with the model?